server/tests/current-user.test.js

```diff
@@ -98,20 +98,25 @@ describe('GraphQL query - Current user', () => {
           id
           schedules {
             id
-            timeSegments {
```

Comment:
sure you want to get rid of this?

Reply:
yes because we cant resolve that yet, so its basically testing for something that wont work